src/components/v5/common/ActionSidebar/partials/MultiSigSidebar/MultiSigSidebar.tsx

```diff
@@ -42,6 +42,7 @@ const MultiSigSidebar: FC<MultiSigSidebarProps> = ({ transactionId }) => {
       <MultiSigWidget
         multiSigData={action.multiSigData}
         actionType={action.type}
```

Comment:
since we've started passing the entire action maybe we can stop passing the action type